### ISSUE TITLE
MyLinks Page: close editor before opening second

### ISSUE THREAD

**Issue by kyberorg** [kyberorg/axe#754]
Steps to reproduce:

0. Save two links
1. Open first editor (by double-clicking description)
2. Open second editor (same way), w/o closing first one.
...
3. Got System Error.

**Comment by kyberorg** [comment]
Fix by adding
```
if (grid.getEditor().isOpen()) grid.getEditor().cancel();
``` 
before 
```
#L296 grid.getEditor().editItem(event.getItem());
```

**Comment by kyberorg** [comment]
Fixed by #755
